app/Resources/views/search/results.html.twig

```diff
@@ -0,0 +1,35 @@
+{% extends 'layout.html.twig' %}
+
+
+{% block content %}
+
+    <div class="row alert alert-info" role="alert">
```

Comment:
I've just updated results page in order to be similar with user's repository page. Some improvements my be added but in an other issue.